analyzers/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -149,25 +161,25 @@ public override void VisitBinaryExpression(BinaryExpressionSyntax node)
                 }
                 else
                 {
-                    base.Visit(node.Left);
-                    base.Visit(node.Right);
+                    Visit(node.Left);
+                    Visit(node.Right);
                 }
             }
 
             private void CheckSpaceBetweenStrings(List<StringWrapper> stringWrappers)
```

Comment:
```suggestion
            private void RaiseIssueIfSpaceExistsBetweenStrings(List<StringWrapper> stringWrappers)
```

Reply:
You mean `RaiseIssueIfSpaceDoesNotExistBetweenStrings` ?